### PR TITLE
adding the context menu to close file tabs with options -closeFile/left/right/closeAll/CloseOthers

### PR DESCRIPTION
**Pull request by AdarshdeepCheema**
Signed-off-by: Adarshdeep Cheema <adarshdeep.cheema@ibm.com>

This PR addresses Issue: https://github.com/zowe/zlux/issues/269

## PR Type
- [ ] Bug fix
- [X] Feature
- [ ] Other (Please indicate)

## PR Checklist
- [x] PR completes `npm run preCommit` without error
- [ ] Relevant Test cases have been added (Unit and or FVT)
- [x] Relevant update to CHANGELOG.md
- [ ] PR from forked repo? Ensure Allow edits by maintaners is set.

### CODE REVIEW

**Review by DivergentEuropeans — COMMENTED** [review]
I think the logic itself looks good, methods make sense 👍 When I first open a tab in JES Explorer, I get some console errors but they don't seem to disrupt anything functionally

image

I'm a little confused about the underlines. Underlines usually tell me we can use hotkeys but I tried some standard hotkeys (shift, alt, ctrl) with those buttons so there's probably a combination in JES that I don't know about. I also don't see any specific code in this PR to buttons like "C" "L" etc. so that's why I'm not sure if they're supposed to be working

Lastly, it wasn't immediately obvious to me "Close to the Left" meant "Close All to the Left"
This is more nitpicky, here's my recommendation for a clearer context menu

Close
Close to the Left
Close to the Right
Close Others
Close All Files

-->

Close
Close All
Close All Except
Close All to the Left
Close All to the Right

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ All notable changes to the JES-Explorer will be documented in this file.
 
 ### New features and enhancements
 - Added the feature to download all files of a JOB in a zip format.
+- Added the different options to close opened file tabs.
```
[the remark this answers is not in the document]
Comment (AdarshdeepCheema) [reply CHANGELOG.md:11]: Addressed all the requested changes

**Comment by AdarshdeepCheema** [comment]
Screen Shot 2021-12-17 at 8 53 07 AM

**Comment by AdarshdeepCheema** [comment]
Screen Shot 2021-12-17 at 8 54 01 AM

**Review by DivergentEuropeans — APPROVED** [review]
Lgtm 🍔 changes addressed

**Pull request merged**